backend/authentication/admin.py

```diff
@@ -3,7 +3,7 @@
 from .models import User
 
 
-class UserAdmin(admin.ModelAdmin):
+class UserAdmin(admin.ModelAdmin[User]):
```

Comment:
I deleted some of the `[Class]` arguments in the merge conflict update as it looks like they weren't used in other places in the PR for #295. Let's keep this in mind and add them back in if necessary. Sorry, just trying to get through all of these and see the new state of things, but we can do a clean up at the end 😊

Reply:
These were added back in in ef1c8a8 :)